hack/boskos.sh

```diff
@@ -85,11 +89,13 @@ account_env_var_file="$(mktemp)"
 
 # Checkout account from Boskos and capture output
 checkout_account "${RESOURCE_TYPE}" 1> "${account_env_var_file}"
+set -o xtrace # Enable debug after checkout
```

Comment:
@Rajalakshmi-Girish, I added this to re-enable debug after checkout as it might give more information in case of error after this point, let me know if this is not correct.